recipes-graphics/libglvnd/libglvnd_1.2.0.bb

```diff
@@ -44,3 +44,5 @@ do_install_append() {
         sed -i -e 's|^Cflags: .*|& -DEGL_NO_X11|g' ${D}${libdir}/pkgconfig/libglvnd.pc ${D}${libdir}/pkgconfig/egl.pc
     fi
 }
+
+RPROVIDES_${PN} += "libegl libgl libgles1 libgles2"
```

Comment:
The mesa recipe does even more than this with an anonymous python function... but it also splits out each library into a separate package, so perhaps this line is enough here.